Automattic/wp-calypso
# chore(build): Show progess during docker builds

This should help us see what's going on during a Docker build. Currently it just blanks out for a few minutes while webpack runs.

#### Testing instructions

* `npm run build-docker` should show progress now during the webpack build
* progress should be shown during a calypso.live build
* `PROGESS=1 npm start` should show progress as well
